README.md

```diff
@@ -228,4 +178,4 @@ Licensed under the [MIT License](./LICENSE).
 
 ## 💜 Thanks
 
-Thanks go out to all of the many sponsors, [Filecoin](https://filecoin.io/), and [Moralis](https://moralis.io/).
+Thanks go out to all of the many sponsors, [Filecoin](https://filecoin.io/)
```

Comment:
This was meant to thank the hackathon sponsors, but maybe not it should just be a general message?

```suggestion
Thanks go out to all of the many sponsors.
```